controller/src/main/java/org/jboss/as/controller/audit/ManagedAuditLoggerImpl.java

```diff
@@ -226,6 +226,7 @@ public void setLoggerStatus(final Status newStatus) {
                     } catch (Exception e) {
                         handleLoggingException(e);
                     }
+                    queuedItems.clear();
```

Comment:
`queuedItems.clear()` inside `for (AuditLogItem record : queuedItems)`
so only one AuditLogItem is written ?